.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,39 +1,57 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: "\U0001F41E Bug report"
+about: Report an issue with Vite
 title: ''
 labels: 'bug: pending triage'
 assignees: ''
 ---
 
-<!--
-中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。
--->
-
-**⚠️ IMPORTANT ⚠️ Please do not ignore this template. If you do, your issue will be closed immediately.**
+<!-- 中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。 -->
+<!-- IMPORTANT: Please do not ignore this template. If you do, your issue will be closed immediately. -->
 
-- [ ] Read [the docs](https://vitejs.dev/guide/).
-- [ ] Make sure this is a Vite issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/vue-next instead.
-- [ ] This is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitejs/vite/discussions) or join our [Discord Chat Server](https://chat.vitejs.dev/).
+### Describe the bug
 
-## Describe the bug
+<!-- A clear and concise description of what the bug is. -->
 
-A clear and concise description of what the bug is.
+<!-- If you intend to submit a PR for this issue, tell us in the description. Thanks! -->
 
-## Reproduction
+### Reproduction
 
-Please provide a link to a repo that can reproduce the problem you ran into.
+<!--
+  Please provide a link to a repo that can reproduce the problem you ran into.
 
-A reproduction is **required** unless you are absolutely sure that the the problem is obvious and the information you provided is enough for us to understand what the problem is. If a report has only vague description (e.g. just a generic error message) and has no reproduction, it will receive "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.
+  A reproduction is required unless you are absolutely sure that the issue is obvious and the provided information is enough to understand the problem. If a report is vague (e.g. just a generic error message) and has no reproduction, it will receive a "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.
+-->
 
-## System Info
+### System Info
 
 - `vite` version:
 - Operating System:
 - Node version:
 - Package manager (npm/yarn/pnpm) and version:
```

Comment:
Consider adding:
> - The output of `npx envinfo --system --npmPackages vite,plugin-vue --binaries --browsers`

Which produces output like this:
```
  System:
    OS: Windows 10 10.0.19042
    CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
    Memory: 14.19 GB / 31.95 GB
  Binaries:
    Node: 14.16.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.6.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 89.0.4389.90
    Edge: Spartan (44.19041.423.0), Chromium (89.0.774.54)
    Internet Explorer: 11.0.19041.1
  npmPackages:
    vite: ^2.1.2 => 2.1.2
```

Are there any other packages we want to check the version of?

Reply:
This looks great, we could add that as a recommendation but I think nobody will do it if it is not required. I would really like this is required, but maybe it is too much. Let's add it with a strong word to try to push people that way at least